usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/DateTime.java

```diff
@@ -121,98 +162,278 @@ private boolean isLeapYear(int year) {
+     */
     public void setMonth(int month) {
         this.month = month;
     }
 
+    /**
+     * Get the day.
+     * @return The day
+     */
     public int getDay() {
         return day;
     }
 
+    /**
+     * Set the day.
+     * @param day The day to set.
+     */
     public void setDay(int day) {
         this.day = day;
     }
 
+    /**
+     * Get the hour.
+     * @return The hour
+     */
     public int getHour() {
         return hour;
     }
 
+    /**
+     * Set the hour.
+     * @param hour The hour to set.
+     */
     public void setHour(int hour) {
         this.hour = hour;
     }
 
+    /**
+     * Get the minutes.
+     * @return The minutes
+     */
     public int getMinutes() {
         return minutes;
     }
 
+    /**
+     * Set the minutes.
+     * @param minutes The minutes to set.
+     */
     public void setMinutes(int minutes) {
         this.minutes = minutes;
     }
 
+    /**
+     * Get the seconds.
+     * @return The seconds
+     */
     public int getSeconds() {
         return seconds;
     }
 
+    /**
+     * Set the seconds.
+     * @param seconds The seconds to set.
+     */
     public void setSeconds(int seconds) {
         this.seconds = seconds;
     }
+
+    /**
+     * Increases a day.
+     */
+    public void increaseDay() {
+        day += 1;
+        if (day > numberOfDays(year, month)) {
+            day = 1;
+            month += 1;
+            if (isOutOfRange(month, hour, minutes, seconds)) {
+                month = 1;
+                year += 1;
+            }
+        }
+    }
+
+    /**
+     * Decreases a day.
+     */
+    public void decreaseDay() {
+        day -= 1;
+        if (day < 1) {
+            month -= 1;
+            day = numberOfDays(year, month);
+            if (month < 1) {
+                year -= 1;
+                month = DECEMBER;
+            }
+        }
+    }
+
+    /**
+     * Check whether the date time is the last week.
+     * @param dateTime The datetime to check
+     * @return True if it is the last week
+     */
+    public static boolean isLastWeek(String dateTime) {
+        boolean result = false;
+        DateTime dateToCheck = new DateTime(dateTime);
+        DateTime currentDate = DateTime.getCurrentDate();
+        for (int i = 0; i <= WEEK_DAYS && !result; ++i) {
+            if (isSameDay(dateToCheck, currentDate)) {
+                result = true;
+            } else {
+                currentDate.decreaseDay();
+            }
+        }
+        return result;
+    }
+
+    /**
+     * Check whether two date time take place on the same day.
+     * @param dateTime The first date time to check
+     * @param currentDate The second date time to check
+     * @return True if the date times take place on the same day
+     */
+    private static boolean isSameDay(DateTime dateTime, DateTime currentDate) {
+        return dateTime.getDay() == currentDate.getDay() && dateTime.getMonth() == currentDate.getMonth() &&
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['&&' should be on a new line.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5351293)